### PR TITLE
Update fingerprints.py Hyundai Tucson 2022 phev

### PR DESCRIPTION
**Pull request by dgcntrk**

### CODE REVIEW

File: selfdrive/car/hyundai/fingerprints.py [hunk 1]
```diff
@@ -1578,13 +1578,25 @@
...
+    (Ecu.hvac, 0x7B0, None): [
+        b"\xf1\x00NX4   97255-P0260CONTROL ASS'Y-DATC  1.00 NX4PHEV ATC 0.7 1.00",
+    ],
+    (Ecu.cornerRadar, 0x7B0, None): [
+        b'\xf1\x003D',
+    ],
+    (Ecu.parkingAdas, 0x7D8, None): [
+        b'\xf1\x87P99910P0900',
+        b'\xf1\x10NX4H ADAS_PRK ANL 1.00 1.00 99910-P0900',
+    ],
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:1599]:
```suggestion
```

File: selfdrive/car/hyundai/fingerprints.py [hunk 2]
```diff
@@ -1578,13 +1578,25 @@
       b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.00 99211-N9260 14Y',
       b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.01 99211-N9100 14A',
       b'\xf1\x00NX4 FR_CMR AT USA LHD 1.00 1.01 99211-N9240 14T',
+      b'\xf1\x00NX4 FR_CMR AT CAN LHD 1.00 1.01 99211-N9100 14A',
+      b'\xf1\x8b !\t\x17',
```
Comment (sshane) [inline selfdrive/car/hyundai/fingerprints.py:1582]:
```suggestion
```

**Comment by sshane** [comment]
Is this your dongle? 3693a6ee0d794ca1

**Comment by dgcntrk** [comment]
> Is this your dongle? 3693a6ee0d794ca1

Yes, that would be me.

**Comment by dgcntrk** [comment]
also in this route: 3693a6ee0d794ca1|2024-02-12--05-48-54--11 at 6am local time I've had a controls lagging issue. I've had another one in the afternoon at a different spot. It seems like I get 1 or 2 for every 20 minutes driven.

**Comment by dgcntrk** [comment]
Today I've got a ecu disconnected error at 3693a6ee0d794ca1|2024-02-15--18-12-54--14  @18:27
and tried to make and as hard turn as possible, the car almost drifted and had to intervene without getting a torque maxed out warning at 3693a6ee0d794ca1|2024-02-15--18-35-20--9 @ 18:44-18:46

**Comment by sshane** [comment]
torqued hasn't learned fully yet, but I found a good section that indicates your max lateral accel is right around what we have in the `params.toml` file, 2.8 m/s^2, so that looks good. Will check out the CAN bus error

**Comment by dgcntrk** [comment]
Any luck on the errors? I can give some more routes with can bus error too if that helps.

**Comment by sshane** [comment]
Can you put the device on WiFi to upload the logs?

**Comment by dgcntrk** [comment]
I would but unfortunately it got bricked this morning. I can only get it to recognize on qdl mode. Waiting a response from support team.

**Comment by sshane** [comment]
Nothing is wrong with the FW version itself, it's just the Canadian version of a US one, so we can merge that now. Reach out if the CAN errors and other device issues keep happening.

| Dongle, platform, VIN                                                                                                             | Name from VIN 🆚 CarInfo                | Segments                  | Bad seg tags                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Good seg tags                                                                          |
|:----------------------------------------------------------------------------------------------------------------------------------|:----------------------------------------|:--------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------|
| [3693a6ee0d794ca1](https://useradmin.comma.ai/?onebox=3693a6ee0d794ca1)<br><sub>HYUNDAI TUCSON 4TH GEN<br>KM8JDDA24........</sub> | HYUNDAI  2022 🆚<br>Hyundai Tucson 2022 | 114 routes, 2113 segments | - [missing ECU FW: 24](https://useradmin.comma.ai/?onebox=3693a6ee0d794ca1%7C2024-02-06--18-37-36)<br>- [spotty FW: 24](https://useradmin.comma.ai/?onebox=3693a6ee0d794ca1%7C2024-02-06--18-37-36)<br>- [VIN spotty (info): 1](https://useradmin.comma.ai/?onebox=3693a6ee0d794ca1%7C2024-03-01--14-19-56)<br>- [CAN invalid: 14](https://useradmin.comma.ai/?onebox=3693a6ee0d794ca1%7C2024-02-16--18-50-48)<br>- [car faulted: 1](https://useradmin.comma.ai/?onebox=3693a6ee0d794ca1%7C2024-02-27--15-37-03) | - valid torque params: 1255<br>- all lat active: 687<br>- no input all lat active: 461 |

**Pull request merged**